src/main/java/calculator/stringCal.java

```diff
@@ -0,0 +1,60 @@
+package calculator;
+
+public class stringCal {
+
+    //string을 반전시키는 함수
+    public static String reverseStr(String str) {
+        String reverse = "";
+        for (int i = str.length() - 1; i >= 0; i--) {
+            reverse = reverse + str.charAt(i);
+        }
+        return reverse;
+    }
+
+    public String numCal(String first, String second){
+        int carry = 0;
```

Comment:
자바에서 메서드명은 동사로 시작합니다!

```suggestion

    public String calculate(String first, String second){
        int carry = 0;
```